apidoc/Identity.yml

```diff
@@ -212,6 +212,23 @@ properties:
     permission: read-only
     platforms: [iphone, ipad, android]
     since: "6.1.0"
+
+  - name: AUTHENTICATION_POLICY_BIOMETRICS_OR_WATCH
+    summary: Device owner was authenticated using a biometric method or the Apple Watch.
+    type: Number
+    permission: read-only
+    platforms: [iphone, ipad]
+    osver: {ios: {min: "13.0"}}
+    since: "8.3.0"
```

Comment:
@hansemannn As we are planning to include this is 8.2.1, can you change since to "8.2.1".